docs/user-guide/concepts/agents/session-management.md

```diff
@@ -0,0 +1,265 @@
+# Session Management
+
+Session management in Strands Agents provides a robust mechanism for persisting agent state and conversation history across multiple interactions. This enables agents to maintain context and continuity even when the application restarts or when deployed in distributed environments.
+
+## Overview
+
+A session represents all of the stateful information that is needed by an agent to function, including:
+
+- Conversation history (messages)
+- Agent state (key-value storage)
+- Other stateful information (like [Conversation Manager](./state.md#conversation-manager))
+
+Strands provides built-in session persistence capabilities that automatically capture and restore this information, allowing agents to seamlessly continue conversations where they left off.
+
+## Basic Usage
+
+First, create an agent with a session manager and use it:
```

Comment:
We have first here, but I'm not seeing it connected to a "then" or anything else